packages/core/src/Modules/Cashier/Components/Error/virtual.jsx

```diff
@@ -15,45 +15,37 @@ class Virtual extends React.Component {
     render = () => {
         return (
             <div className='cashier__wrapper'>
-                <ThemedScrollbars
-                    style={{ position: 'absolute', height: 'calc(100% - 4.8rem)', width: 'calc(100% - 4.8rem)' }}
-                    autoHide
-                    autoHeightMax={550}
-                >
-                    <div className='cashier__virtual-center'>
-                        <h2 className='cashier-error__header cashier__virtual-header'>
-                            <Localize i18n_default_text={"You're currently using a demo account"} />
-                        </h2>
-                        {this.props.has_real_account ? (
-                            <React.Fragment>
-                                <p className='cashier__paragraph cashier__text'>
-                                    <Localize i18n_default_text='You need to switch to a real money account to use this feature.' />
-                                    <br />
-                                    <Localize i18n_default_text='You can do this by selecting a real account from the Account Switcher.' />
-                                </p>
-                                <div className='cashier__account-switch-icon' />
-                            </React.Fragment>
-                        ) : (
-                            <React.Fragment>
-                                <p className='cashier__paragraph cashier__text'>
-                                    <Localize
-                                        i18n_default_text={
-                                            "You need a real money account to use this feature. It's easy to create a real money account and start trading."
-                                        }
-                                    />
-                                </p>
-                                <Button
-                                    className='cashier-error__button'
-                                    has_effect
-                                    text={localize('Create my real account')}
-                                    onClick={this.onClickSignup}
-                                    primary
-                                    large
-                                />
-                            </React.Fragment>
-                        )}
-                    </div>
-                </ThemedScrollbars>
+                <h2 className='cashier-error__header cashier__virtual-header'>
+                    <Localize i18n_default_text={"You're currently using a demo account"} />
```

Comment:
did translation already handled single quote inside double quotes in deriv app?

Reply:
I was under the impression that it does cover (I wrote it to support that) but I just tested and it doesn't work. Let's follow that up in another PR.